lib/ndr_import/file/docx.rb

```diff
@@ -9,19 +10,35 @@ module File
     # This class is a modern Word document file handler that returns a single table.
     # It only works on .docx documents
     class Docx < Base
+      include OfficeFileHelper
+
       private
 
       def rows(&block)
         return enum_for(:rows) unless block
 
-        doc = ::Docx::Document.open(SafeFile.safepath_to_string(@filename))
+        doc = if @options.key?(:file_password)
```

Comment:
Not strictly related to this PR, but according to the documentation, using `open` without a block won't close the underlying file:

>  With no associated block, Docx::Document.open is a synonym for Docx::Document.new. If the optional code block is given, it will be passed the opened +docx+ file as an argument and the Docx::Document object will automatically be closed when the block terminates. The values of the block will be returned from Docx::Document.open.

What's less clear is that the `Docx::Document` class doesn't appear to provide a `close` method - I think you'd have to do `doc.zip.close`, which seems a little clunky.

Reply:
The library's own `README` examples don't seem to care, so probably not worth giving more than a passing thought to.